includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -668,11 +662,13 @@ public function get_proxy_setup_url( $access_code = '', $error_code = '' ) {
 	 * Gets the list of features to declare support for when setting up with the proxy.
 	 *
 	 * @since 1.1.0
+	 * @since n.e.x.t Added 'credentials_retrieval'
 	 * @return array
```

Comment:
Per docs guidelines, there needs to be an empty line before any param/return docs. Also, every parameter and return value should have a description, here the latter is missing.

Reply:
Can we bake these rules into the PHPCS config? These should be caught by tooling rather than code review 😄

Reply:
Agreed. We would need to check if these kinds of rules exist somewhere though, or whether we'd need to formalize them ourselves (which would make it quite some work). WP core doesn't enforce this anywhere unfortunately, but still this is how it should be done. Potentially there is even a path for contributing something back.

Feel free to open an issue for this, I'd definitely support that. It's just that, given the size of that issue and other priorities, it might be a while.

Reply:
Yeah, I'll open an issue for this 👍  Understood regarding the priority vs bandwidth.

In the meantime, can we document all of these exceptions somewhere?

Reply:
I opened https://github.com/google/site-kit-wp/issues/924 and assigned to you

In the meantime, can we document all of these exceptions somewhere? 🙂